table.css

```
@@ -0,0 +1,23 @@
+.table {
+  border: 1px solid black;
+  width: 100%;
+  height: 100%;
+}
+
+.table__headerRow {
```

```suggestion
.table__header-row {
```

Apply this to all css classes please.